src/ray/common/scheduling/scheduling_ids.h

```diff
@@ -0,0 +1,50 @@
+#ifndef RAY_COMMON_SCHEDULING_SCHEDULING_IDS_H
+#define RAY_COMMON_SCHEDULING_SCHEDULING_IDS_H
+
+#include <iostream>
+#include <map>
```

Comment:
not needed any more (now using unordered_map)

Reply:
removed it